rust/db_handling/src/identities.rs

```diff
@@ -100,6 +100,40 @@ lazy_static! {
     static ref REG_PATH: Regex = Regex::new(r"^(?P<path>(//?[^/]+)*)(///(?P<password>.+))?$").expect("known value");
 }
 
+#[derive(Clone, Encode, Decode)]
+pub enum SignaturesBulk {
+    #[codec(index = 4)]
+    V1(SignaturesBulkV1),
```

Comment:
@pgolovkin this is essentially the type that is proposed to use to provide bulks of signatures to external apps. The current variant starts with the byte `0x04` in order not to conflict with the existing format of exporting single signatures which is essentially `MultiSignature::encode()` and would start with bytes in set `[0x00, 0x01, 0x02]` for respective signature scheme variants in `MultiSignature`.